src/app/components/core/configuration-manager/configuration-group/configuration-group.component.ts

```diff
@@ -239,21 +239,25 @@ export class ConfigurationGroupComponent implements AfterViewInit {
         e.group === 'Basic' ? acc.unshift(e) : acc.push(e);
         return acc;
       }, []);
+
+    // set advance as a first tab if no default config
+    if (this.groups.length == 0) {
+      this.selectedGroup = groups[0]?.group
```

Comment:
if length is zero then how it comes  groups[0]

Reply:
global vs local group. Changed the name of local group to dynamicGroup for clear understanding